eval/normalize.py

```
@@ -0,0 +1,97 @@
+
+def normalize_quorum(
+    data: list | dict,
+    tool_name: str,
+    file_path: str | None = None,
+) -> list[CanonicalFinding]:
+    findings = []
+    if isinstance(data, dict):
+        # {file_path: [findings...]} grouped format
+        for fp, file_findings in data.items():
+            for f in file_findings:
+                findings.append(_quorum_finding(f, tool_name, fp))
```

Ruff (0.15.12)

[warning] Use `list.extend` to create a transformed list

(PERF401)